prime-router/docs/how-to-onboard-a-receiver.md

```diff
@@ -41,16 +52,24 @@ Below is an example of the organization file
         receivingFacilityOID:
 ```
 
-In the above example, the jurisdictional filter searches the `ordering_facility_state` field in the report for anything that matches the code LT.
+In the above example, the jurisdictional filter searches the `ordering_facility_state` field in the report for anything 
+that matches the code LT. Filters can be applied to the organization or receiver. For more information on filters see: 
+(https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/playbooks/how-to-use-filters.md)
 
-In addition, there is the translation section, which specifies the output format that will be sent to the receiver. Currently, we have three formats available:
+In addition, there is the translation section, which specifies the output format that will be sent to the receiver. 
+Currently, we have three formats available:
 
-- HL7 - Each record is converted to HL7 before being sent to the receiver
-- CSV - The records are converted to CSV before being sent to the receiver
+- HL7
+- CSV
+- FHIR
 
-The quality filters verify that test results have the minimum fields required by most public health jurisdictions. Those fields are made adjustable to be customized per jurisdictional specifications. The default filter includes these requirements:
+The quality filters verify that test results have the minimum fields required by most public health jurisdictions. 
+Those fields are made adjustable to be customized per jurisdictional specifications. These filters are applied by 
+default to the receiver and set in code. They do not need to be added to the receiver setting. If a custom rule is 
```

Comment:
`If a custom rule is added to the quality filter for a receiver, all default quality filters will be ignored.`
Wait, is this actually true? If I add a custom filter for Patient First Name, it will no longer require Device Identifier to have a value?

Reply:
yes that is true